spec/support/shared_examples/an_entity_mapper_update_operation.rb

```diff
@@ -0,0 +1,3 @@
+RSpec.shared_examples_for 'an entity mapper update operation' do |relation|
+  it { expect { entity_mapper.import }.to change { relation::Operation.where(operation: 'U').count }.by(1) }
+end
```

Comment:
We might merge all these entity mapper operations in just one file, and also add the 
"an xml node and mapper that are not a destroy operation"

Reply:
I prefer the top level explicitness and the modular approach gives us the ability to have different setups for each depending on the mapper